collectors/auth0/auth0_collector.js

```diff
@@ -50,22 +51,14 @@ class Auth0Collector extends PawsCollector {
             clientSecret: collector.secret,
             scope: 'read:logs'
         });
```

Comment:
Could you please change https://github.com/alertlogic/paws-collector/pull/116/files#diff-b5ce20b57b16e2d88fa5da3430cbc695R47 to `const hostname = collector.pawsDomainEndpoint;`?

Reply:
Done